.gitignore

```diff
@@ -1,3 +1,14 @@
 obj/
 compile_commands.json
 *.pyc
+
+# Files that should be written from CAN spec
+can_validator/fsae_can_spec.h
+MY17_CAN_Library.c
+MY17_CAN_Library.h
+ids.h
+bms.h
+can_node.h
+current_sensor.h
+dash.h
+vcu.h
```

Comment:
Append [this](https://github.com/github/gitignore/blob/master/Python.gitignore), since you now have Python project in this repo.

Reply:
(Added in 14a86a3, not marked as outdated because I didn't edit the particular line I think)